Readme.md

```diff
@@ -32,6 +32,7 @@ Add your dependencies in your `Brewfile`:
 ```ruby
 cask_args appdir: "/Applications"
 tap "homebrew/cask"
+tap "homebrew/cask", pin: true
```

Comment:
@mikesplain Sorry for pedantry but I think it's worth adding a tap here that would make sense to pin instead (as pinning a cask tap won't really do anything)? Perhaps just the `telemachus` one below?